corehq/apps/api/tests/test_user_updates.py

```diff
@@ -45,23 +42,25 @@ def test_update_password_without_strong_passwords_succeeds(self):
 
         try:
             update(self.user, 'password', 'abc123')
-        except ValidationError:
+        except UpdateUserException:
             self.fail('Unexpected ValidationError raised.')
```

Comment:
nit
```suggestion
            self.fail('Unexpected UpdateUserException raised.')
```
An alternate suggestion would be to remove the try/except and possibly add a comment:
```py
update(self.user, 'password', 'abc123')  # should not raise
```